TOC-tidb-cloud.md

```diff
@@ -18,65 +18,72 @@
+      - [PyMySQL](/develop/dev-guide-sample-application-python-pymysql.md)
+      - [SQLAlchemy](/develop/dev-guide-sample-application-python-sqlalchemy.md)
+      - [peewee](/develop/dev-guide-sample-application-python-peewee.md)
   - Third-Party Support
     - [Third-Party Tools Supported by TiDB](/develop/dev-guide-third-party-support.md)
     - [Known Incompatibility Issues with Third-Party Tools](/develop/dev-guide-third-party-tools-compatibility.md)
+  - Development Reference
+    - Design Database Schema
+      - [Overview](/develop/dev-guide-schema-design-overview.md)
+      - [Create a Database](/develop/dev-guide-create-database.md)
+      - [Create a Table](/develop/dev-guide-create-table.md)
+      - [Create a Secondary Index](/develop/dev-guide-create-secondary-indexes.md)
+    - Write Data
+      - [Insert Data](/develop/dev-guide-insert-data.md)
+      - [Update Data](/develop/dev-guide-update-data.md)
+      - [Delete Data](/develop/dev-guide-delete-data.md)
+      - [Periodically Delete Expired Data Using TTL (Time to Live)](/time-to-live.md)
+      - [Prepared Statements](/develop/dev-guide-prepared-statement.md)
+    - Read Data
+      - [Query Data from a Single Table](/develop/dev-guide-get-data-from-single-table.md)
+      - [Multi-Table Join Queries](/develop/dev-guide-join-tables.md)
+      - [Subquery](/develop/dev-guide-use-subqueries.md)
+      - [Paginate Results](/develop/dev-guide-paginate-results.md)
+      - [Views](/develop/dev-guide-use-views.md)
+      - [Temporary Tables](/develop/dev-guide-use-temporary-tables.md)
+      - [Common Table Expression](/develop/dev-guide-use-common-table-expression.md)
+      - Read Replica Data
+        - [Follower Read](/develop/dev-guide-use-follower-read.md)
+        - [Stale Read](/develop/dev-guide-use-stale-read.md)
+      - [HTAP Queries](/develop/dev-guide-hybrid-oltp-and-olap-queries.md)
+      - [FastScan](/develop/dev-guide-use-fastscan.md)
+    - Transaction
+      - [Overview](/develop/dev-guide-transaction-overview.md)
+      - [Optimistic and Pessimistic Transactions](/develop/dev-guide-optimistic-and-pessimistic-transaction.md)
+      - [Transaction Restraints](/develop/dev-guide-transaction-restraints.md)
+      - [Handle Transaction Errors](/develop/dev-guide-transaction-troubleshoot.md)
+    - Optimize
+      - [Overview](/develop/dev-guide-optimize-sql-overview.md)
+      - [SQL Performance Tuning](/develop/dev-guide-optimize-sql.md)
+      - [Best Practices for Performance Tuning](/develop/dev-guide-optimize-sql-best-practices.md)
+      - [Best Practices for Indexing](/develop/dev-guide-index-best-practice.md)
+      - Other Optimization Methods
+        - [Avoid Implicit Type Conversions](/develop/dev-guide-implicit-type-conversion.md)
+        - [Unique Serial Number Generation](/develop/dev-guide-unique-serial-number-generation.md)
+    - [Connection Pools and Connection Parameters](/develop/dev-guide-connection-parameters.md)
```

Comment:
Because "Connection Pools and Connection Parameters" is for Java users, can we make it the last doc in Connect to TiDB Cloud > Java?

Reply:
done.